R/bs-theme-layers.R

```diff
@@ -108,25 +109,23 @@ ensure_default_flag <- function(x) {
 #' @param rules Sass rules.
 #' @export
 bs_add_rules <- function(theme, rules) {
-  bs_add_layers(theme, sass_layer(rules = rules))
+  bs_add_bundles(theme, sass_layer(rules = rules))
 }
 
 #' @rdname bs_add_variables
 #' @param declarations Sass functions and mixins.
 #' @export
 bs_add_declarations <- function(theme, declarations) {
-  bs_add_layers(theme, sass_layer(declarations = declarations))
+  bs_add_bundles(theme, sass_layer(declarations = declarations))
 }
 
 #' @rdname bs_add_variables
```

Comment:
Add a `@describeIn` explaining the `...` and remove the (trailing) explanation from `bs_add_variables()`